go.mod

```diff
@@ -134,3 +134,27 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.2 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace (
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.31.2
```

Comment:
why the packages were replaced to older versions here?

Reply:
Oh sorry, the changes are made before the module bump

Reply:
udpated

Reply:
BTW this change is to solve the "invalid version: unknown revision v0.0.0" problem following https://stackoverflow.com/questions/59187781/revision-v0-0-0-unknown-for-go-get-k8s-io-kubernetes. It cannot work without these replacements when I'm developing in Goland.

Reply:
removed this change after discussion